vision.md

```diff
@@ -206,26 +206,188 @@ The framework includes modules for cloud connectivity, data mapping, device
 management, intra-edge communication, and certificate management, all aspects
 and challenges our target personas need to address.
 
+We therefore define the following requirements.
```

Comment:
I would consider to move these requirements in a separate documents. These are derived form the vision and more subject to changes.

Reply:
No matter where we keep it, we should have reference of that to this document.

Reply:
The vision would be a single paragraph. This document also goes into the strategy of how to achieve the vision and is not a pure vision statement. I believe it is very important to describe these key problems and application scenarios/use-cases here as the whole value of thin-edge.io is based on them and they should be well understood by the community.

Reply:
I think we have to types of content that needs to move:
1. the non functional requirements I would move to the technical vision ( design principles doc
2. the functional aspects and everythin that you could call feature I would put into a more abstracted way into the "roadmap.md"

Reply:
> the functional aspects and everythin that you could call feature I would put into a more abstracted way into the "roadmap.md"

Why not in a [concept overview](https://github.com/thin-edge/thin-edge.io/issues/1416) document?

I think that using the documentation to convey our vision is a good way to be consistent (avoiding to repeat ourselves) and clear (with a single source of truth).